### PR TITLE
Issue 496

### PR DESCRIPTION
**Pull request by tcnichol**
I have changed the json for fixed metadata.

First, if the metadata has changed over time there will be keys 'start_dates'. Each start date will have the complete json for that date.

Second, right now a start date has to be sent - it is not 'smart' in the sense of recognizing that the json has start dates because even if we know that, we would need a date to know which metadata to apply. 

Please let me know what other things need to change. Pull request for issue 497 changes things in getting the sensor fixed metadata, so please review together.

### CODE REVIEW

File: sensors/SWIR/sensor_fixed_metadata.json [hunk 1]
```diff
@@ -1,22 +1,82 @@
...
+    "sensor_description": "hyperspectral camera to measure shortwave infrared (SWIR) radiation",
+    "sensor_purpose": "measures spectral reflectance from 950nm to 2500nm",
+    "location_in_gantry_system": "camera box, facing ground",
+    "location_in_camera_box_m": {
+      "x": "0.877",
+      "y": "2.325",
+      "z": "0.635"
+    }
+  },
+  "2012-01-01": {
```
Comment (max-zilla) [inline sensors/SWIR/sensor_fixed_metadata.json]: looks like 2012-01-01 gets repeated a couple times here.

**Comment by tcnichol** [comment]
I am not sure about one of the metadata changes (see commit log) so I'd appreciate someone going over this and the .xml files for the new metadata.

**Comment by max-zilla** [comment]
We discussed the FOV for VNIR and SWIR during last week meeting:
- VNIR reports FOV Y = 1.000m, but we convert that to 21 degressand will keep the same.
- SWIR reports FOV Y = 0.75m, we convert to 44.5 degrees

https://github.com/terraref/sensor-metadata/commit/2ebf65ee809cf678fc55bc6da321faf01326e52d#diff-5ad8a7fd73ed7273a44f804e1eeb6b4d

Both are OK.

**Comment by max-zilla** [comment]
I double-checked all and compared to XML files, looks good. Will merge at same time we merge https://github.com/terraref/terrautils/pull/56 (waiting for @craig-willis to review)

**Review by craig-willis — APPROVED** [review]

**Pull request merged**